Dynamo20/Dynamo_UI/Dynamo20_UI.csproj

```diff
@@ -275,13 +275,13 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <PostBuildEvent>rd /S /Q "C:\Users\$(Username)\AppData\Roaming\Dynamo\Dynamo Core\2.0\packages\BHoM"
-xcopy /Y /E /I C:\ProgramData\BHoM\Assemblies\*.dll "C:\Users\$(Username)\AppData\Roaming\Dynamo\Dynamo Core\2.0\packages\BHoM\bin"
+xcopy /Y /E /I C:\ProgramData\BHoM\Assemblies\*.dll "C:\Users\$(Username)\AppData\Roaming\Dynamo\Dynamo Core\2.0\packages\BHoM\bin" /EXCLUDE:$(ProjectDir)excludes.txt
```

Comment:
```suggestion
xcopy /Y /E /I C:\ProgramData\BHoM\Assemblies\*.dll "C:\Users\$(Username)\AppData\Roaming\Dynamo\Dynamo Core\2.0\packages\BHoM\bin" /EXCLUDE:"$(ProjectDir)excludes.txt"
```

Reply:
Yeah it fails to read excludes.txt if the path is wrapped in "" - that was my original idea but was failing as described by you. Not sure what to do then?